pages/vi/profiles/RyanDu.md

```diff
@@ -0,0 +1,21 @@
+# Ryan Du
+
+Location | Timezone | OS
+:--- | :--- | :---
+Dallas, TX | CST | MacOS Mojave 10.14.6|
```

Comment:
please check githack page. this code has extra cell

Reply:
Hi, I've already fix the problem